Doc/library/optparse.rst

```diff
@@ -991,6 +993,8 @@ one that makes sense for *all* options.
 Option attributes
 ^^^^^^^^^^^^^^^^^
 
+.. class:: Option
```

Comment:
This is basically the same situation as above as its just a stub without a clear or explicit description, and presumably should be handled the same way (one way or the other):

Add an explicit summary description,

```suggestion
.. class:: Option

   A single command line argument,
   with various attributes passed by keyword to the constructor.
   Normally created with :meth:`OptionParser.add_option` rather than directly,
   and can be overridden by a custom class via the *option_class* argument
   to :class:`OptionParser`.     
```

Or just not add it for now and handle it in a followup:

```suggestion
```

Alternatively, or additionally, we _could_ indent the following paragraphs and attributes under the class (eliminating the extra `Option` from the attribute names), as is standard for other classes, but that would be a big delta for this PR, so is probably best considered for a followup.